video/out/vo_gpu_next.c

```diff
@@ -841,7 +841,11 @@ static void apply_target_options(struct priv *p, struct pl_frame *target)
     int dither_depth = opts->dither_depth;
     if (dither_depth == 0) {
         struct ra_swapchain *sw = p->ra_ctx->swapchain;
-        dither_depth = sw->fns->color_depth ? sw->fns->color_depth(sw) : 0;
+        if (sw->fns->color_depth) {
```

Comment:
For this to work, you would need to remove dummy implementation from vulkan. Good thing to do either way. https://github.com/mpv-player/mpv/blob/cbe30f614d878c2622e86d9c5cd8da558ef16936/video/out/vulkan/context.c#L312-L315

Reply:
Done